pyomo/contrib/pyros/tests/test_config.py

```diff
@@ -557,16 +557,22 @@ def test_positive_int_or_minus_one(self):
         """
         Test positive int or -1 validator works as expected.
         """
-        standardizer_func = PositiveIntOrMinusOne()
+        standardizer_func = positive_int_or_minus_one
         self.assertIs(
```

Comment:
This should really be `assertEqual` and not `assertIs`

Reply:
I originally used `assertIs` in lieu of `assertEqual` to ensure `positive_int_or_minus_one` casts a numeric type object to an `int`, instead of just returning the object; `assertEqual` would admit a comparison of `1` with `1.00`, for example. Are you suggesting that `assertEqual` is surely sufficient for this check? If so, I will proceed with the change you have suggested. Otherwise, the check in line 569, which uses `assertEqual` and should otherwise be similar to the check here, may also need to be changed.

Reply:
The problem is that there is no guarantee that `int(1) is int(1)`.  I think that if you want to check both the type and the value, I would recommend:
```python
ans = standardizer_func(1.0)
self.assertEqual(ans, 1)
self.assertIs(type(ans), int)
```